src/main/java/org/jabref/model/entry/identifier/DOI.java

```diff
@@ -167,6 +170,7 @@ public static Optional<DOI> parse(String doi) {
             LatexToUnicodeFormatter formatter = new LatexToUnicodeFormatter();
             String cleanedDOI = doi;
             cleanedDOI = URLDecoder.decode(cleanedDOI, StandardCharsets.UTF_8);
+            cleanedDOI = cleanedDOI.replaceAll("\\^", "");
```

Comment:
Please add a comment:

    // needs to be handled before LatexToUnicode, because otherwise `^` will be treated as conversion superscript